core/src/com/po/fuck/Assets.java

```diff
@@ -1,40 +1,27 @@
 package com.po.fuck;
 
 import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.assets.AssetDescriptor;
 import com.badlogic.gdx.assets.AssetManager;
 import com.badlogic.gdx.graphics.Texture;
-import com.badlogic.gdx.graphics.g2d.Animation;
-import com.badlogic.gdx.graphics.g2d.TextureRegion;
-import com.po.fuck.view.GifDecoder;
+import com.badlogic.gdx.utils.JsonReader;
+import com.badlogic.gdx.utils.JsonValue;
+
 
 public class Assets {
+    private static JsonReader jsonReader = new JsonReader();
 
-    public AssetManager manager = new AssetManager();
-    public static final AssetDescriptor<Texture> glock3Texture =
-            new AssetDescriptor<>("glock3.png", Texture.class);
-    public static final AssetDescriptor<Texture> glock2Texture =
-            new AssetDescriptor<>("glock2.png", Texture.class);
-    public static final AssetDescriptor<Texture> basicEnemyTexture =
-            new AssetDescriptor<>("basicEnemy.png", Texture.class);
-    public static final AssetDescriptor<Texture> bulletTexture =
-            new AssetDescriptor<>("bullet2.png", Texture.class);
-    public static final AssetDescriptor<Texture> playerTexture =
-            new AssetDescriptor<>("FUCKerWithoutHands2.png", Texture.class);
-    public static final AssetDescriptor<Texture> laserGunTexture =
-            new AssetDescriptor<>("laser_gun2.png", Texture.class);
+    public static AssetManager manager = new AssetManager();
 
-    public static final AssetDescriptor<Texture> islandTexture =
-            new AssetDescriptor<>("island2.png", Texture.class);
+    public static JsonValue getInfo(String name) {
+        return jsonReader.parse(Gdx.files.internal("assetsData.json")).get(name);
```

Comment:
This function should cache the data, not read it every time.

Reply:
@MAKMED1337  Is it will be okey if I rename it into ```getAssetObjectInfo```?